common/segments/root.xml

```diff
@@ -28,10 +28,14 @@ For terms of use, see http://www.unicode.org/copyright.html
 				<variable id="$LV">\p{Grapheme_Cluster_Break=LV}</variable>
 				<variable id="$LVT">\p{Grapheme_Cluster_Break=LVT}</variable>
 				<!-- Note: The following may overlap with the above -->
-				<variable id="$Virama">[\p{Gujr}\p{sc=Telu}\p{sc=Mlym}\p{sc=Orya}\p{sc=Beng}\p{sc=Deva}&amp;\p{Indic_Syllabic_Category=Virama}]</variable>
-				<variable id="$LinkingConsonant">[\p{Gujr}\p{sc=Telu}\p{sc=Mlym}\p{sc=Orya}\p{sc=Beng}\p{sc=Deva}&amp;\p{Indic_Syllabic_Category=Consonant}]</variable>
+				<!-- Note: ConjunctLinkingScripts is not used anymore, instead that list exists in the derivation of Indic_Conjunct_Break. -->
+				<!-- It is kept here so that the diff of the generated test cases compared to the Unicode 15.1 β is minimal. -->
+				<!-- TODO(egg): Consider removing in Unicode 16.0. -->
+				<variable id="$ConjunctLinkingScripts">[\p{Gujr}\p{sc=Telu}\p{sc=Mlym}\p{sc=Orya}\p{sc=Beng}\p{sc=Deva}]</variable>
+				<variable id="$ConjunctLinker">\p{Indic_Conjunct_Break=Linker}</variable>
+				<variable id="$LinkingConsonant">\p{Indic_Conjunct_Break=Consonant}</variable>
 				<variable id="$ExtPict">\p{Extended_Pictographic}</variable>
-				<variable id="$ExtCccZwj">[[$Extend-\p{ccc=0}] $ZWJ]</variable>
+				<variable id="$ExtCccZwj">[\p{Indic_Conjunct_Break=Linker}\p{Indic_Conjunct_Break=Extend}]</variable>
```

Comment:
Hmm. Should $ExtCccZwj still include $ZWJ? If so, where does that get added? If not, maybe $ExtCccZwj should renamed...

Reply:
Scripsit @macchiati:
I think [ZWJ] is in InCB=Extend.
Indeed, see https://github.com/unicode-org/unicodetools/blob/9595f090650e99e3e752b37a7a3866ac8a91999b/unicodetools/src/main/java/org/unicode/text/UCD/ToolUnicodePropertySource.java#L1193-L1203.